katello/helper.rb

```diff
@@ -1,7 +1,7 @@
 module KatelloUtilities
   module Helper
     def last_scenario
-      File.basename(File.readlink("/etc/foreman-installer/scenarios.d/last_scenario.yaml")).split(".")[0]
+      File.basename(File.readlink("/etc/foreman-installer/scenarios.d/last_scenario.yaml")).split(".")[0] rescue 'katello'
```

Comment:
is there a way we can avoid a rescue? Maybe check if the file exists?

Reply:
Yeah, that might be possible as well.